lifelines/statistics.py

```diff
@@ -208,7 +208,7 @@ def chisq_test(U, degrees_freedom, alpha):
 
 
 def two_sided_z_test(Z, alpha):
-    p_value = 1 - max(stats.norm.cdf(Z), 1 - stats.norm.cdf(Z))
+    p_value = 1 - np.max(stats.norm.cdf(Z), 1 - stats.norm.cdf(Z))
```

Comment:
Investigating this speed up, I see `max` being faster: 
![screen shot 2015-02-04 at 08 30 44](https://cloud.githubusercontent.com/assets/884032/6041204/20301b96-ac48-11e4-9530-bf707af76046.png)